omg/completion/bash.py

```diff
@@ -10,15 +10,15 @@
 }
 
 _omg_completionetup() {
-    local COMPLETION_OPTIONS=""
+    local COMPLETION_OPTIONS="-o default"
     local BASH_VERSION_ARR=(${BASH_VERSION//./ })
     # Only BASH version 4.4 and later have the nosort option.
     if [ ${BASH_VERSION_ARR[0]} -gt 4 ] || ([ ${BASH_VERSION_ARR[0]} -eq 4 ] && [ ${BASH_VERSION_ARR[1]} -ge 4 ]); then
-        COMPLETION_OPTIONS="-o nosort -o default"
+        COMPLETION_OPTIONS="-o nosort"
```

Comment:
@bostrt So now if bash version is 4.4 we will only set "-o nosort". Doesn't that mean that now "-o default" will not be set if the bash version is 4.4?